gnosis/cowswap
# Hotfix/v1.0.1

# Summary

Closes #1151


Light mode:
![image](https://user-images.githubusercontent.com/2352112/129358943-b5d3d427-ad89-4805-90a3-c5d8f874c0f4.png)


Dark mode:
![image](https://user-images.githubusercontent.com/2352112/129359009-a845b6bc-cd3e-43bb-a612-317ae017e7a5.png)


# Out of scope
* Add a header to navigate to the home
* Nicer styles, we should do in `develop` a new PR using a cow Meme


# To Test
I've made the Cow Game page to throw an error so we can see the app crashing.

1. Try to play the game (use the menu to navigate)
2. Observe the error page, make sure there's no traces of Uniswap
